### PR TITLE
Fix/tao 3251 item responses

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TAO-3251

Fix the behaviour of the shortcut that allows to check a choice (SPACE key).

Without this fix, when you press the space bar to check a response, the radio button is checked, then immediately unchecked

### CODE REVIEW

File: views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js [hunk 1]
```diff
@@ -55,13 +55,16 @@ define([
...
         $choiceInputs.on('keydown.commonRenderer', function(e){
+            var $this = $(this);
             var keyCode = e.keyCode ? e.keyCode : e.charCode;
             if(keyCode !== KEY_CODE_TAB){
                 e.preventDefault();
             }
 
             if( keyCode === KEY_CODE_SPACE || keyCode === KEY_CODE_ENTER){
-                _triggerInput($(this).closest('.qti-choice'));
+                _.delay(function(){
```
Comment (ssipasseuth) [inline views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js]: might be useful to add a comment for future developer who will see this code

Comment (jsconan) [inline views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js]: This PR  is only intended to fix the `space` key selection. The tab navigation has been made inside another PR, related to the test runner only.

**Comment by jsconan** [comment]
This PR  is only intended to fix the `space` key selection. The tab navigation has been made inside another PR, related to the test runner only.

File: views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js (hunk 1 above)
Comment (jsconan) [inline views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js]: Sure!

**Comment by ssipasseuth** [comment]
This is good enough for now but does not solve the real issue. The real issue is that somehow the click event callback is executed when the focus is set on an input and space key is pressed. We need to investigate this further.

**Pull request merged**